docker-compose-examples/grafana-monitoring/.env

```diff
@@ -10,3 +10,5 @@ PROMETHEUS_CONF=./conf/prometheus.yml
 GRAFANA_CONF=./conf/grafana/grafana.ini
 GRAFANA_PROVISIONING=./conf/grafana/provisioning
 GRAFANA_PORT=3000
+PROMETHEUS_PORT=9090
+LOKI_PORT=3100
```

Comment:
do we need those? I'd rather not allow external access to the backends for the toy setup?

Reply:
> do we need those? I'd rather not allow external access to the backends for the toy setup?

reverted. I had different thought and exposed it there too.